llvm/test/MachineVerifier/test_g_vscale.mir

```diff
@@ -0,0 +1,16 @@
+# RUN: not --crash llc -verify-machineinstrs -run-pass none -o /dev/null %s 2>&1 | FileCheck %s
+# REQUIRES: aarch64-registered-target
```

Comment:
The target isn't specified here, and this doesn't need one as written

Reply:
Updated